shell/platform/darwin/common/framework/Headers/FlutterBaseDartProject.h

```diff
@@ -0,0 +1,53 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_FLUTTERBASEDARTPROJECT_H_
+#define FLUTTER_FLUTTERBASEDARTPROJECT_H_
+
+#import <Foundation/Foundation.h>
+
+#import "FlutterMacros.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * A set of Flutter and Dart assets used by a `FlutterEngine` to initialize execution.
+ *
+ * TODO(stuartmorgan): Align API with iOS DartProject, and combine.
+ */
+@interface FlutterBaseDartProject : NSObject
```

Comment:
The FlutterDartProject is completely copy pasted here. 
The only difference is that I removed FLUTTER_DARWIN_EXPORT from the original `FlutterDartProject`

Reply:
Why remove the visibility macro? We'll need it to make the symbols public.

Reply:
Ah I see - you've left it on the subclasses so it shouldn't be necessary for LTO, since there's already a reference to it within the framework.